redaction/draft-strad-trans-redaction.md

```diff
@@ -187,14 +300,14 @@ in the certificate, TLS clients MUST also:
 
 # Security Considerations
 
-## Avoiding Overly Redacting Domain Name Labels
+## Avoiding Overly Redacted Domain Names
 
```

Comment:
Please add a TODO(eranm): here to describe harm to the CT ecosystem that would arise from the use of redaction being too widespread.